### ISSUE TITLE
Apply SCImageView filter on a view

### ISSUE THREAD

**Issue by maxs15** [rFlex/SCRecorder#175]
Hey guys,
I was wondering if it is possible to apply a filter on top of a view that contains a video and other stuff.
I tried without any success, any help would be appreciated :)

**Comment by rFlex** [comment]
I'm gonna add a new way to process a video through the UIKit API. I will
let you know once it's done ;).

**Comment by maxs15** [comment]
awesome :)

**Comment by maxs15** [comment]
Any update on that new awesome way to process a video through the UIKit API ?
I'm currently working on a SCRecorder wrapper for react-native [react-native-screcorder](https://github.com/maxs15/react-native-screcorder).
With this new feature I would be able to process the filter directly on top of the current react-native player [react-native-video](https://github.com/brentvatne/react-native-video).

Cheers :beers:

**Comment by rFlex** [comment]
Yes it's done! Use the "overlay" property. The video will always be fully sized in the background though. I haven't yet made a way to put the video container within a complex view tree.

**Comment by maxs15** [comment]
Cool, I will check that !
Thanks
